src/main/java/com/sarahhand/fractals/json/ConfigSaverLoader.java

```diff
@@ -60,13 +61,13 @@ public FractalConfig load(Class<? extends FractalConfig> target, String fileName
 	 * @param stream
 	 * @return
 	 */
-	public FractalConfig load(Class<? extends FractalConfig> target, InputStream stream){
+	public FractalConfig load(InputStream stream){
 
 		JsonReaderWriter readerWriter = new JsonReaderWriter(PropertyNamingStrategy.SNAKE_CASE);
 
-		FractalConfig config = readerWriter.<FractalConfig>read(stream, target);
+		FractalEnvelope envelope = readerWriter.read(stream);
 
-		return config;
+		return envelope.getConfig();
```

Comment:
Can readerWriter.read(stream) return null?  If so, you should check if envelope is null before returning envelope.getConfig().